### PR TITLE
MAINTAINERS: add @Nico385412 to collaborators

### PR DESCRIPTION
**Pull request by Nico385412**
Issue https://github.com/tldr-pages/tldr/issues/8224.

As ask in the issue here is the PR to add myself to the list of collaborators.
I've read the [maintainer's guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/maintainers-guide.md) And i'll do my best 😄  

Thank you again !

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -84,6 +84,8 @@ If you are an owner of the organization, you can see an automated list
   [19 May 2021](https://github.com/tldr-pages/tldr/issues/5993) - [21 June 2021](https://github.com/tldr-pages/tldr/issues/6149)
 - Emily Grace Seville ([@EmilySeville7cfg](https://github.com/EmilySeville7cfg)):
   [19 January 2022](https://github.com/tldr-pages/tldr/issues/1209#issuecomment-285924778) — [24 April 2022](https://github.com/tldr-pages/tldr/issues/8053)
+- Nicolas Hansse ([@Nico385412](https://github.com/nico385412)):
```
Comment (navarroaxel) [inline MAINTAINERS.md:87]:
```suggestion
- Nicolas Hansse ([@Nico385412](https://github.com/Nico385412)):
```

**Review by navarroaxel — COMMENTED** [review]
Your entry should be placed after `258204` member and before `Owen Voke` in the list.

File: MAINTAINERS.md [hunk 2]
```diff
@@ -52,6 +52,8 @@ If you are an owner of the organization, you can see an automated list
   [23 Nov 2021](https://github.com/tldr-pages/tldr/issues/7404) — present
 - **258204 ([@258204](https://github.com/258204))**:
   [10 December 2021](https://github.com/tldr-pages/tldr/issues/7522) — present
+- Nicolas Hansse ([@Nico385412](https://github.com/Nico385412)):
```
Comment (navarroaxel) [inline MAINTAINERS.md:55]: and in bold
```suggestion
- **Nicolas Hansse ([@Nico385412](https://github.com/Nico385412))**:
```

Comment (Nico385412) [reply MAINTAINERS.md]: done ! 
Sorry i didn't understood at first the time that `**....**` was for the actual maintainers 🤦

**Review by CleanMachine1 — APPROVED** [review]
Thanks for all of your contributions!

**Pull request merged**